otel-extension/src/main/scala/io/scalac/mesmer/otelextension/instrumentations/akka/http/Instrumentation.scala

```diff
@@ -0,0 +1,196 @@
+package io.scalac.mesmer.otelextension.instrumentations.akka.http
+
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation
+import io.opentelemetry.javaagent.extension.instrumentation.TypeTransformer
+import net.bytebuddy.asm.Advice
+import net.bytebuddy.description.`type`.TypeDescription
+import net.bytebuddy.description.method.MethodDescription
+import net.bytebuddy.implementation.SuperMethodCall
+import net.bytebuddy.matcher.ElementMatcher
+import net.bytebuddy.matcher.ElementMatchers
+
+import io.scalac.mesmer.instrumentation.http.impl.OverridingRawPathMatcher
+
+object Instrumentation {
```

Comment:
maybe rename to "PathAttributeExtractors" (or similar)? "Instrumentation" to me sounds like you will attach some "instruments" here. We just add advice with VirtualFields setters/getters. WDYT? 

Also might be confusing when we add a new HTTP instrumentation (connections counter that is pending)

Reply:
Hmm, I see it as a collection of `TypeInstrumentations`, so I'g go with `PathMatchingInstrumentations` or something similar